redbot/cogs/audio/apis/__init__.py

```diff
@@ -0,0 +1,11 @@
+from . import (
+    api_utils,
+    global_db,
+    interface,
+    local_db,
+    local_tracks,
```

Comment:
`local_tracks` needs to be removed.
Also, I have a question - why isn't this file just empty? Python should automatically resolve the `redbot.core.audio.apis.submodule` imports.
```suggestion
    local_db,
```